### PR TITLE
Stop running tests on Windows Vista and Windows 8 (fixes issue #588)

### PR DESCRIPTION
**Pull request by whimboo**
This PR will fix issue #588 by removing all machines and updating the pulse config files.

### CODE REVIEW

**Comment by whimboo** [comment]
@AutomatedTester can you please review so we can land this ASAP? Thanks

File: config/staging/jenkins.patch [hunk 1]
```diff
@@ -217,7 +217,7 @@ index f32806a..51c5c79 100644
        <mode>NORMAL</mode>
        <retentionStrategy class="hudson.slaves.RetentionStrategy$Always"/>
        <launcher class="hudson.slaves.JNLPLauncher"/>
--      <label>windows xp vista 7 8 8.1 32bit 64bit endurance</label>
+-      <label>windows xp 7 8.1 32bit 64bit endurance</label>
 -      <nodeProperties/>
 -      <userId>anonymous</userId>
 +      <label>windows 8.1 64bit endurance</label>
```
Comment (davehunt) [inline config/staging/jenkins.patch]: Do we still run endurance? I suspect this label can go, but that would be a separate patch.

**Comment by davehunt** [comment]
r+

File: config/staging/jenkins.patch (hunk 1 above)
Comment (whimboo) [inline config/staging/jenkins.patch]: No we don't. I will file a separate issue for it. Thanks.
